spec/spec_helper_acceptance_local.rb

```diff
@@ -0,0 +1,190 @@
+services:
+  compose_test:
+    image: *default-image
+    command: /bin/sh -c "while true; do echo hello world; sleep 1; done"
+        EOS
+    docker_compose_content_v3_windows = <<-EOS
+version: "3"
+services:
+  compose_test:
+    image: winamd64/hello-seattle
+    command: cmd.exe /C "ping 8.8.8.8 -t"
+networks:
+  default:
+    external:
+      name: nat
+      EOS
+    docker_compose_override_v3_windows = <<-EOS
+version: "3"
+services:
+  compose_test:
+    image: winamd64/hello-seattle:nanoserver
+    command: cmd.exe /C "ping 8.8.8.8 -t"
+networks:
+  default:
+    external:
+      name: nat
+      EOS
+    docker_compose_override_v3_windows2016 = <<-EOS
+version: "3"
+services:
+  compose_test:
+    image: winamd64/hello-seattle:nanoserver-sac2016
+    command: cmd.exe /C "ping 8.8.8.8 -t"
+networks:
+  default:
+    external:
+      name: nat
+      EOS
+    docker_stack_content_windows = <<-EOS
+version: "3"
+services:
+  compose_test:
+    image: winamd64/hello-seattle
+    command: cmd.exe /C "ping 8.8.8.8 -t"
+      EOS
+    docker_stack_override_windows = <<-EOS
+version: "3"
+services:
+  compose_test:
+    image: winamd64/hello-seattle:nanoserver
+      EOS
+    docker_stack_override_windows2016 = <<-EOS
+version: "3"
+services:
+  compose_test:
+    image: winamd64/hello-seattle:nanoserver-sac2016
+      EOS
+    if os[:family] == 'windows'
+      create_remote_file(host, '/tmp/docker-compose-v3.yml', docker_compose_content_v3_windows)
+      create_remote_file(host, '/tmp/docker-stack.yml', docker_stack_content_windows)
+      if os[:release] =~ %r{2019}
+        create_remote_file(host, '/tmp/docker-compose-override-v3.yml', docker_compose_override_v3_windows)
+        create_remote_file(host, '/tmp/docker-stack-override.yml', docker_stack_override_windows)
+      else
+        create_remote_file(host, '/tmp/docker-compose-override-v3.yml', docker_compose_override_v3_windows2016)
+        create_remote_file(host, '/tmp/docker-stack-override.yml', docker_stack_override_windows2016)
+      end
+    else
+      create_remote_file(host, '/tmp/docker-compose-v3.yml', docker_compose_content_v3)
+      create_remote_file(host, '/tmp/docker-stack.yml', docker_compose_content_v3)
+      create_remote_file(host, '/tmp/docker-compose-override-v3.yml', docker_compose_override_v3)
+      create_remote_file(host, '/tmp/docker-stack-override.yml', docker_stack_override_v3)
+    end
+
+    next unless os[:family] == 'windows'
+    result = run_shell("ipconfig | findstr /i 'ipv4'")
+    raise 'Could not retrieve ip address for Windows box' if result.exit_code != 0
+    ip = result.stdout.split("\n")[0].split(':')[1].strip
+    retry_on_error_matching(60, 5, %r{connection failure running}) do
+      @windows_ip = ip
+    end
+    apply_manifest("class { 'docker': docker_ee => true, extra_parameters => '\"insecure-registries\": [ \"#{@windows_ip}:5000\" ]' }", catch_failures: true)
+    docker_path = '/cygdrive/c/Program Files/Docker'
```

Comment:
Do we still have a dependency on Cygwin?

Reply:
we should not